gensim/models/keyedvectors.py

```diff
@@ -196,14 +253,16 @@ def add(self, entities, weights, replace=False):
             self.vectors[in_vocab_idxs] = weights[in_vocab_mask]
 
     def __setitem__(self, entities, weights):
-        """Add entities and theirs vectors in a manual way.
+        """
+        Add entities and theirs vectors in a manual way.
+
         If some entity is already in the vocabulary, old vector is replaced with the new one.
-        This method is alias for `add` with `replace=True`.
+        This method is alias for :meth:`~add()` with `replace=True`.
```

Comment:
I'm not sure that current link works correctly (typically need full path) like `~gensim.models.keyedvectors.BaseKeyedVectors.add`